test/integration/test_build.py

```diff
@@ -148,6 +148,7 @@ def test_has_pytz(self, cli, container_runtime, pytz):
         r = cli(f'{container_runtime} run --rm {ee_tag} pip3 show pytz')
         assert 'World timezone definitions, modern and historical' in r.stdout, r.stdout
 
+    @pytest.mark.skip("Concurrency issues plus output parsing is using the incorrect format")
```

Comment:
@samdoran for future, it's better to use xfail per https://pganssle-talks.github.io/xfail-lightning/ + https://pypi.org/p/pytest-rerunfailures exists.

Reply:
If it's flaky, use `xfail` + `strict=False` unless you want retries.